### ISSUE TITLE
Ruby Together style grouping

### ISSUE THREAD

**Issue by ericholscher** [pybee/paying-the-piper#36]
I really like the work going on at https://rubytogether.org/ -- which lets companies give money to the foundation that then gets doled out to community projects based on need. I see this as only really sustaining fundamental projects in the ecosystem, and not a solution for smallish projects.

In the Python world, the PSF does a small part of this job with grants, but it isn't nearly as focused on the goal of sustaining engineering. I'd love to have something like this be able to fund larger resources like PyPI, pip, Read the Docs, PyPy, CPython and other large core infrastructure.

It seems like a lot of us are doing legwork to raise funding independently, and merging these efforts to fund more projects would save us all a lot of work. Even just getting 1 person who could handle fundraising and then distribute funds to the projects, who could focus on doing work instead of raising money, would be a huge win.

**Comment by seiyria** [comment]
I agree here! As it stands right now, I'm looking to fund a project, but to do that, I'm going to need to do a whole bunch of business-y stuff to get there... it's offputting to me, I just want to make things.

**Comment by nayafia** [comment]
I really like Ruby Together, too! Agreed on wanting to put heads together to combine resources and person-power. Count me in as someone who likes to do the "business-y stuff". :)

**Comment by nanuxbe** [comment]
+1
Anyone from the PSF on this thread to give feedback about feasibility with them?

**Comment by ariddell** [comment]
I think this is a great idea. I'd be willing to give money to a centrally managed fund where money was given to projects that needed it. I worry a little bit about the necessary administrative overhead of this. It looks like RubyTogether has some experience with this though. I like the monthly reports.

**Comment by nayafia** [comment]
Yeah, RubyTogether makes it work because the admin = grantees. As far as I can tell, they're essentially raising money for themselves, a team of two, to work on open source full-time.

Funds where admin is giving to third-party grantees could theoretically do some sort of management or overhead fee, like nonprofits and VCs do.

**Comment by ariddell** [comment]
From the website it looks like the Board (7 people) selects who gets money. And the Board is elected by 97 developers (who each pay >=$40 a month). This seems reasonable. Monthly income is $10k.

**Comment by nayafia** [comment]
My understanding is the Board helps select which projects to work on, but the funding goes to the core RubyTogether team (Andre + David) to work on the projects themselves, right? Which is slightly different from funding outside contributors who want to work on, say, Bundler. Still a really cool model and interesting approach to the problem.

**Comment by ariddell** [comment]
Interesting. Well, it certainly would be a more compelling example (to me, at least) if it could fund outside contributors to work on other Ruby projects (like Bundler).

I'm still impressed. $10k a month is nothing to sneeze at.

**Comment by nayafia** [comment]
Yes, I'm interested in facilitating that model as well. To @ericholscher's point I'd like to see funding go towards "smallish projects" - stuff that's not quite on Bundler level but still very important.
